components/clm/src/biogeophys/SoilWaterMovementMod.F90

```diff
@@ -490,8 +491,13 @@ subroutine soilwater_zengdecker2009(bounds, num_hydrologyc, filter_hydrologyc, &
          j = nlev2bed(c)
          if(jwt(c) == nlevbed) then 
             tempi = 1._r8
-            temp0 = (((sucsat(c,j)+zwtmm(c)-zimm(c,j))/sucsat(c,j)))**(1._r8-1._r8/bsw(c,j))
-            vol_eq(c,j+1) = -sucsat(c,j)*watsat(c,j)/(1._r8-1._r8/bsw(c,j))/(zwtmm(c)-zimm(c,j))*(tempi-temp0)
+	    delta_z_zwt= zwtmm(c)-zimm(c,j)
```

Comment:
Please indent the code

Reply:
@bishtgautam ,  the indent is correct when I open the file in  my folder for open using the online editing tool . This is only view bug from github. Let me know if that is not the case  for you.